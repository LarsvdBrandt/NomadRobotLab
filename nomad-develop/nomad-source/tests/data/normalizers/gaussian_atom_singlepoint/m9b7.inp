%mem=12000000
#ROHF/6-311G*  scf=verytight

Helium Atom cation He+
casno=7440597
method=9
basis=7

1 2
He


