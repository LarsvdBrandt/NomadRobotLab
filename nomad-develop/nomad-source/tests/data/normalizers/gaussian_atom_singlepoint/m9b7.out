 Entering Gaussian System, Link 0=g03
 Initial command:
 /usr/local/apps/g03b05/g03/l1.exe /wrk/rdj3/Gau-21602.inp -scrdir=/wrk/rdj3/
 Entering Link 1 = /usr/local/apps/g03b05/g03/l1.exe PID=     21234.
  
 Copyright (c) 1988,1990,1992,1993,1995,1998,2003, Gaussian, Inc.
                  All Rights Reserved.
  
 This is the Gaussian(R) 03 program.  It is based on the
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under DFARS:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, duplication or disclosure by the US Government is subject
 to restrictions as set forth in subparagraph (c)(1)(ii) of the
 Rights in Technical Data and Computer Software clause at DFARS
 252.227-7013.
  
 Gaussian, Inc.
 Carnegie Office Park, Building 6, Pittsburgh, PA 15106 USA
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is subject
 to restrictions as set forth in subparagraph (c) of the
 Commercial Computer Software - Restricted Rights clause at FAR
 52.227-19.
  
 Gaussian, Inc.
 Carnegie Office Park, Building 6, Pittsburgh, PA 15106 USA
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 03, Revision B.05,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, J. A. Montgomery, Jr., T. Vreven, 
 K. N. Kudin, J. C. Burant, J. M. Millam, S. S. Iyengar, J. Tomasi, 
 V. Barone, B. Mennucci, M. Cossi, G. Scalmani, N. Rega, 
 G. A. Petersson, H. Nakatsuji, M. Hada, M. Ehara, K. Toyota, 
 R. Fukuda, J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, 
 H. Nakai, M. Klene, X. Li, J. E. Knox, H. P. Hratchian, J. B. Cross, 
 C. Adamo, J. Jaramillo, R. Gomperts, R. E. Stratmann, O. Yazyev, 
 A. J. Austin, R. Cammi, C. Pomelli, J. W. Ochterski, P. Y. Ayala, 
 K. Morokuma, G. A. Voth, P. Salvador, J. J. Dannenberg, 
 V. G. Zakrzewski, S. Dapprich, A. D. Daniels, M. C. Strain, 
 O. Farkas, D. K. Malick, A. D. Rabuck, K. Raghavachari, 
 J. B. Foresman, J. V. Ortiz, Q. Cui, A. G. Baboul, S. Clifford, 
 J. Cioslowski, B. B. Stefanov, G. Liu, A. Liashenko, P. Piskorz, 
 I. Komaromi, R. L. Martin, D. J. Fox, T. Keith, M. A. Al-Laham, 
 C. Y. Peng, A. Nanayakkara, M. Challacombe, P. M. W. Gill, 
 B. Johnson, W. Chen, M. W. Wong, C. Gonzalez, and J. A. Pople, 
 Gaussian, Inc., Pittsburgh PA, 2003.
 
 *************************************************
 Gaussian 03:  IBM64-RS6000-G03RevB.05  8-Nov-2003
                    18-Aug-2009 
 *************************************************
 %mem=12000000
 ---------------------------
 #ROHF/6-311G* scf=verytight
 ---------------------------
 1/38=1/1;
 2/17=6,18=5,40=1/2;
 3/5=4,6=6,7=1,11=2,16=1,25=1,30=1/1,2,3;
 4/7=6/1;
 5/5=2,17=3,32=1,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,9=1/99;
 -----------------------------------------------------
 Helium Atom cation He+ casno=7440597 method=9 basis=7
 -----------------------------------------------------
 Symbolic Z-matrix:
 Charge =  1 Multiplicity = 2
 He
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic     Atomic              Coordinates (Angstroms)
 Number     Number      Type              X           Y           Z
 ---------------------------------------------------------------------
    1          2             0         .000000     .000000     .000000
 ---------------------------------------------------------------------
 Stoichiometry    He(1+,2)
 Framework group  OH[O(He)]
 Deg. of freedom     0
 Full point group                 OH      NOp  48
 Largest Abelian subgroup         D2H     NOp   8
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic     Atomic              Coordinates (Angstroms)
 Number     Number      Type              X           Y           Z
 ---------------------------------------------------------------------
    1          2             0         .000000     .000000     .000000
 ---------------------------------------------------------------------
 Standard basis: 6-311G(d) (5D, 7F)
 There are     3 symmetry adapted basis functions of AG  symmetry.
 There are     0 symmetry adapted basis functions of B1G symmetry.
 There are     0 symmetry adapted basis functions of B2G symmetry.
 There are     0 symmetry adapted basis functions of B3G symmetry.
 There are     0 symmetry adapted basis functions of AU  symmetry.
 There are     0 symmetry adapted basis functions of B1U symmetry.
 There are     0 symmetry adapted basis functions of B2U symmetry.
 There are     0 symmetry adapted basis functions of B3U symmetry.
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
     3 basis functions,     5 primitive gaussians,     3 cartesian basis functions
     1 alpha electrons        0 beta electrons
       nuclear repulsion energy          .0000000000 Hartrees.
 NAtoms=    1 NActive=    1 NUniq=    1 SFac= 1.00D+00 NAtFMM=   60 Big=F
 One-electron integrals computed using PRISM.
 NBasis=     3 RedAO= T  NBF=     3     0     0     0     0     0     0     0
 NBsUse=     3 1.00D-06 NBFU=     3     0     0     0     0     0     0     0
 Harris functional with IExCor=  205 diagonalized for initial guess.
 ExpMin= 2.45D-01 ExpMax= 9.81D+01 ExpMxC= 9.81D+01 IAcc=1 IRadAn=         1 AccDes= 1.00D-06
 HarFok:  IExCor= 205 AccDes= 1.00D-06 IRadAn=         1 IDoV=1
 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 Initial guess orbital symmetries:
       Occupied  (A1G)
       Virtual   (A1G) (A1G)
 The electronic state of the initial guess is 2-A1G.
 Warning!  Cutoffs for single-point calculations used.
 Requested convergence on RMS density matrix=1.00D-04 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-02.
 Requested convergence on             energy=5.00D-05.
 No special actions if energy rises.
 Keep R1 and R2 integrals in memory in canonical form, NReq=      819107.
 SCF Done:  E(ROHF) =  -1.99813924037     A.U. after    7 cycles
             Convg  =     .1041D-04             -V/T =  1.9996
             S**2   =    .7500
 Annihilation of the first spin contaminant:
 S**2 before annihilation      .7500,   after      .7500

 **********************************************************************

            Population analysis using the SCF density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A1G)
       Virtual   (A1G) (A1G)
 The electronic state is 2-A1G.
 Alpha  occ. eigenvalues --    -.91664
 Alpha virt. eigenvalues --     .23970   5.31039
          Condensed to atoms (all electrons):
              1
     1  He   1.000000
 Mulliken atomic charges:
              1
     1  He   1.000000
 Sum of Mulliken charges=   1.00000
 Atomic charges with hydrogens summed into heavy atoms:
              1
     1  He   1.000000
 Sum of Mulliken charges=   1.00000
          Atomic-Atomic Spin Densities.
              1
     1  He   1.000000
 Mulliken atomic spin densities:
              1
     1  He   1.000000
 Sum of Mulliken spin densities=   1.00000
 Electronic spatial extent (au):  <R**2>=      .7560
 Charge=     1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=      .0000    Y=      .0000    Z=      .0000  Tot=      .0000
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=     -.3389   YY=     -.3389   ZZ=     -.3389
   XY=      .0000   XZ=      .0000   YZ=      .0000
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=      .0000   YY=      .0000   ZZ=      .0000
   XY=      .0000   XZ=      .0000   YZ=      .0000
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=      .0000  YYY=      .0000  ZZZ=      .0000  XYY=      .0000
  XXY=      .0000  XXZ=      .0000  XZZ=      .0000  YZZ=      .0000
  YYZ=      .0000  XYZ=      .0000
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=     -.1106 YYYY=     -.1106 ZZZZ=     -.1106 XXXY=      .0000
 XXXZ=      .0000 YYYX=      .0000 YYYZ=      .0000 ZZZX=      .0000
 ZZZY=      .0000 XXYY=     -.0369 XXZZ=     -.0369 YYZZ=     -.0369
 XXYZ=      .0000 YYXZ=      .0000 ZZXY=      .0000
 N-N= 0.000000000000D+00 E-N=-3.997044063797D+00  KE= 1.998904822533D+00
 Symmetry AG   KE= 1.998904822533D+00
 Symmetry B1G  KE= 0.000000000000D+00
 Symmetry B2G  KE= 0.000000000000D+00
 Symmetry B3G  KE= 0.000000000000D+00
 Symmetry AU   KE= 0.000000000000D+00
 Symmetry B1U  KE= 0.000000000000D+00
 Symmetry B2U  KE= 0.000000000000D+00
 Symmetry B3U  KE= 0.000000000000D+00
                          Isotropic Fermi Contact Couplings
        Atom                 a.u.       MegaHertz       Gauss      10(-4) cm-1
     1  He(3)              2.32420   -7914.37937   -2824.04723   -2639.95280
 --------------------------------------------------------
       Center         ----  Spin Dipole Couplings  ----
                      3XX-RR        3YY-RR        3ZZ-RR
 --------------------------------------------------------
     1   Atom         .000000       .000000       .000000
 --------------------------------------------------------
                        XY            XZ            YZ
 --------------------------------------------------------
     1   Atom         .000000       .000000       .000000
 --------------------------------------------------------


 ---------------------------------------------------------------------------------
              Anisotropic Spin Dipole Couplings in Principal Axis System
 ---------------------------------------------------------------------------------

       Atom             a.u.   MegaHertz   Gauss  10(-4) cm-1        Axes

              Baa      .0000      .000      .000      .000  1.0000   .0000   .0000
     1 He(3)  Bbb      .0000      .000      .000      .000   .0000  1.0000   .0000
              Bcc      .0000      .000      .000      .000   .0000   .0000  1.0000
 

 ---------------------------------------------------------------------------------

 1\1\GINC-HAL4\SP\ROHF\6-311G(d)\He1(1+,2)\RDJ3\18-Aug-2009\0\\#ROHF/6-
 311G* SCF=VERYTIGHT\\Helium Atom cation He+ casno=7440597 method=9 bas
 is=7\\1,2\He\\Version=IBM64-RS6000-G03RevB.05\State=2-A1G\HF=-1.998139
 2\RMSD=1.041e-05\Dipole=0.,0.,0.\PG=OH [O(He1)]\\@


 EDUCATION DOESN'T COST 
           ... IT PAYS!

 -- POSTMARK, UNIV. OF MISSISSIPPI, DEC 78
 Job cpu time:  0 days  0 hours  0 minutes  1.9 seconds.
 File lengths (MBytes):  RWF=     13 Int=      0 D2E=      0 Chk=      9 Scr=      1
 Normal termination of Gaussian 03 at Tue Aug 18 17:11:35 2009.
